test/e2e/resources/function_event_invoke_config.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: lambda.services.k8s.aws/v1alpha1
+kind: Function
+metadata:
+  name: $FUNCTION_NAME
+  annotations:
+    services.k8s.aws/region: $AWS_REGION
+spec:
+  name: $FUNCTION_NAME
+  code:
+    s3Bucket: $BUCKET_NAME
+    s3Key: $LAMBDA_FILE_NAME
+  functionEventInvokeConfig:
+    destinationConfig:
+      onSuccess:
+        destination: $ON_SUCCESS_DESTINATION
+      onFailure:
+        destination: $ON_FAILURE_DESTINATION
+    maximumEventAgeInSeconds: $MAXIMUM_EVENT_AGE_IN_SECONDS
+    maximumRetryAttempts: $MAXIMUM_RETRY_ATTEMPTS
+  role: $LAMBDA_ROLE
+  runtime: python3.9
+  handler: main
+  description: function created by ACK lambda-controller e2e tests
+  reservedConcurrentExecutions: $RESERVED_CONCURRENT_EXECUTIONS
+  codeSigningConfigARN: "$CODE_SIGNING_CONFIG_ARN"
```

Comment:
If you're not using this fields you can just remove them and remove the replacements in the pytest as well

Reply:
Done